packages/controls/src/widget_string.ts

```diff
@@ -671,7 +671,7 @@ export class ComboboxView extends TextView {
       o.value = v;
       optionFragment.appendChild(o);
     }
-    this.datalist.appendChild(optionFragment);
+    this.datalist.replaceChildren(...optionFragment.children);
```

Comment:
Would it not be faster (and more consistent) with `this.datalist.replaceChildren(optionFragment)` ?